scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -97,11 +97,13 @@
 axis_charge = common.axis_charge
 axis_passIso = common.axis_passIso
 axis_passMT = common.axis_passMT
+axis_mt = hist.axis.Variable([0,4,10,11,15,20,21,40] + list(range(45, 100, 5)) + [100, 110, 120, 130, 150, 200], name = "mt", underflow=False, overflow=True)
```

Comment:
How did you choose the binning at low mT? With quantiles? I was particularly puzzled by those two bins between 10 and 11, or 20 and 21.
For the high mT region, for the purpose of the fakes I think you could safely stop at 120 to save 3 bins of memory, and keep the rest in the overflow, because there aren't really many fakes expected for so large mT